src/gui/widgets/MixerChannelLcdSpinBox.cpp

```diff
@@ -69,5 +79,25 @@ void MixerChannelLcdSpinBox::contextMenuEvent(QContextMenuEvent* event)
 	contextMenu->exec(QCursor::pos());
 }
 
+void MixerChannelLcdSpinBox::enterValue()
+{
+	bool ok;
+	int new_val;
+
+	new_val = QInputDialog::getInt(
+			this, tr("Set value"),
+			tr("Please enter a new value between %1 and %2:").
+			arg(model()->minValue()).
+			arg(model()->maxValue()),
+			model()->value(),
+			model()->minValue(),
+			model()->maxValue(),
+			model()->step<int>(), &ok);
+
+	if(ok)
+	{
+		model()->setValue(new_val);
+	}
```

Comment:
I think this is better for readability, inlining the `model()->*` calls where a bit verbose.
```suggestion
	const auto val = model()->value();
	const auto min = model()->minValue();
	const auto max = model()->maxValue();
	const auto step = model()->step<int>();
	const auto label = tr("Please enter a new value between %1 and %2:").arg(min).arg(max);

	auto ok = false;
	const auto newVal = QInputDialog::getInt(this, tr("Set value"), label, val, min, max, step, &ok);

	if (ok) { model()->setValue(newVal); }
```